Public/components/cards/cards.css

```diff
@@ -0,0 +1,92 @@
+@import url("/Public/components/alert/alert.css");
+.basic-card{
+    width:fit-content;
+    border-radius: 8px;
+    background-color: white;
```

Comment:
to maintain uniformity, use "Var" instead of giving specific colors & hard coding it.